content-node/docker-compose/README.md

```diff
@@ -2,71 +2,58 @@
 
 This repository contains the necessary files to deploy a Content Node using Docker Compose.
 
-## Prerequisites
+This README assumes you have followed the [Content Node Setup Guide](https://docs.earthfast.com/node-operators/content-node-setup) and satisfied all prerequisites & registered your Content Node.
 
-- A Linux-based system (preferably Ubuntu)
-- An internet-connected server with a static public IP address
-- A DNS record pointing to your server's IP address eg `content1.us-central1-a.testnet-sepolia.earthfastnodes.com`
-- A Registered Content Node --> https://docs.earthfast.com/node-operators/content-node-setup#register-your-nodes
+## Setup
 
-## Quick Start
-
-#### 1. Clone this repository and run the setup script:
+Clone this repository and run the setup script:
 ```sh
-git clone https://github.com/earthfast/earthfast-node-setup-examples.git
-cd earthfast-node-setup-examples/content-node/docker-compose
-./setup.sh
-```
-
-You can use the `--staging` flag for sepolia staging instead.
+> git clone https://github.com/earthfast/node-operator-tooling
+> cd node-operator-tooling/content-node/docker-compose
+> ./setup.sh
 ```
-./setup.sh --staging
-```
-
-##### SSL Configuration
 
-If `SETUP_SSL` is set to 'true', the setup will automatically configure SSL using Let's Encrypt. Make sure the url from `SERVER_NAME` correctly pointed to your server's IP address before enabling this option.
+The `setup.sh` script will prompt you for environment variables:
+* `SERVER_NAME` = your node's FQDN
+* `NODE_ID` = the NodeID outputted in Step 5 above
```

Comment:
lets add a link to this? https://docs.earthfast.com/node-operators/content-node-setup#register-your-node-s-onchain

Reply:
whoops, fixed